src/placeos-rest-api/controllers/modules.cr

```diff
@@ -65,14 +65,14 @@ module PlaceOS::Api
           query.filter({"driver_id" => [driver_id]})
         end
 
-        if (connected = args.connected)
+        unless (connected = args.connected).nil?
```

Comment:
I have never seen this pattern before. Why is assignment put as a condition in an if/else cause? Does it return a boolean value?

Reply:
It's a tight (albeit indirect) way of doing an assignment.
Assignment expressions in crystal return the value of the assignment, so you get away with what's happening here.